src/frontend/apps/impress/src/features/docs/doc-table-content/components/Heading.tsx

```diff
@@ -47,23 +50,23 @@ export const Heading = ({
         }
 
         editor.setTextCursorPosition(headingId, 'end');
+
         document.querySelector(`[data-id="${headingId}"]`)?.scrollIntoView({
           behavior: 'smooth',
-          block: 'start',
+          inline: 'center',
+          block: 'center',
         });
       }}
+      $radius="4px"
+      $background={isActive ? `${colorsTokens()['greyscale-100']}` : 'none'}
       $css="text-align: left;"
     >
       <Text
-        $theme="primary"
-        $padding={{ vertical: 'xtiny', left: 'tiny' }}
-        $size={sizeMap[level]}
+        $width="100%"
+        $padding={{ vertical: 'xtiny', left: leftPaddingMap[level] }}
+        $variation={isActive ? '1000' : '700'}
+        $weight={isActive ? 'bold' : 'normal'}
```

Comment:
Long titles are out of container.

![image](https://github.com/user-attachments/assets/ec4e3524-e9de-4dd1-9f3b-5c18d84c1746)

```suggestion
        $weight={isActive ? 'bold' : 'normal'}
        $css="overflow-wrap: break-word;" 
```

Or ellipsis ?